### DISCUSSION TITLE
text model share community like civitai

### DISCUSSION THREAD

**Discussion by newcworld** [oobabooga/text-generation-webui#2512] [Show and tell]
Hi guys. Now you can share text model on https://cworld.ai/
It's a model share community
you can share and explore different model 

Also I provide more detailed doc for get start
https://docs.cworld.ai/docs/intro
</br>
**online demo** 
[Reddit Crush Post generate](https://docs.cworld.ai/tutorial-app/reddit-simplify)
**reddit data download**
For Reddit data search and download https://docs.cworld.ai/dataset/reddit

**Comment by luthis1124** [comment]
I'm really keen to see how this develops. Bookmarked!
